Transfer/Sources/TransferSourceCell.swift

```diff
@@ -89,23 +89,25 @@ extension TransferSourceCell {
                   additionalInfo: transferSourceCellConfiguration.additionalText,
                   currency: transferSourceCellConfiguration.destinationCurrency,
                   availableBalance: transferSourceCellConfiguration.availableBalance,
-                  fontIcon: transferSourceCellConfiguration.fontIcon.rawValue)
+                  fontIcon: transferSourceCellConfiguration.fontIcon.rawValue,
+                  type: transferSourceCellConfiguration.type)
     }
 
     private func configure(title: String,
                            additionalInfo: String? = nil,
                            currency: String?,
                            availableBalance: String?,
-                           fontIcon: String) {
+                           fontIcon: String,
+                           type: TransferSourceType) {
         textLabel?.text = title
         textLabel?.adjustsFontForContentSizeCategory = true
         textLabel?.numberOfLines = 0
         textLabel?.lineBreakMode = .byWordWrapping
         textLabel?.accessibilityIdentifier = "transferSourceTitleLabel"
-        detailTextLabel?.text = additionalInfo
-        detailTextLabel?.numberOfLines = 0
-        detailTextLabel?.adjustsFontForContentSizeCategory = true
-        detailTextLabel?.lineBreakMode = .byWordWrapping
+        detailTextLabel?.text = type == .user ? currency : additionalInfo
+        detailTextLabel?.adjustsFontSizeToFitWidth = true
+        detailTextLabel?.minimumScaleFactor = 0.1
+        detailTextLabel?.numberOfLines = 2
```

Comment:
What happens when the user has just prepaid card balance?

Reply:
Prepaid card details does not get affected as they will be in 2 lines only.